Nodejs/Product/Nodejs/Options/NodejsNpmOptionsPage.cs

```diff
@@ -47,6 +48,8 @@ public NodejsNpmOptionsPage()
         /// </summary>
         public bool ShowOutputWindowWhenExecutingNpm { get; set; }
 
+        public bool EnableAutomaticTypingsAcquisition { get; set; }
```

Comment:
@mousetraps Just to be super clear, for option two, that would be: remove the setting in the UI all together?